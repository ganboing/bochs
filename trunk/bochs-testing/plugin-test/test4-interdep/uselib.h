extern int n_operations;
