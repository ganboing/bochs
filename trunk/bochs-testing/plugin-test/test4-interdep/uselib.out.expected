start
at first, n_operations = 0
Module name is 'AddModule'
module2: operation_occurred
operate(5,12) = 17
stop
at end, n_operations = 1
